src/sysv/systemd/sssd_check_socket_activated_responders.py.in

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  SSSD
+#  sssd_check_socket_activated_responder.py
```

Comment:
I think this is the wrong approach. This drags in a python dependency for the `sssd-common` package, which is a huge increase in the footprint for it (and could make SSSD unsuitable for constrained use-cases like in appliances).

Please reimplement this in C, using libini_config.